Show beta label on topic and subtopic pages

tijmenb opened the pull request:
Users of Collections Publisher can mark a topic `beta`, which means the topic is not finished yet. 

Trello: https://trello.com/c/H9UMp051/139-allow-topic-subtopics-to-be-labelled-as-beta

Previous PRs: 

- https://github.com/alphagov/govuk-content-schemas/pull/77
- https://github.com/alphagov/collections-publisher/pull/67
- https://github.com/alphagov/collections-api/pull/20

## Before

![screen shot 2015-05-28 at 13 59 04](https://cloud.githubusercontent.com/assets/233676/7860529/d15c6fd0-0541-11e5-8750-464ddfaa17c4.png)
![screen shot 2015-05-28 at 13 58 56](https://cloud.githubusercontent.com/assets/233676/7860514/c694b7f6-0541-11e5-979c-ced78c8062a9.png)

## After

![screen shot 2015-05-28 at 13 58 44](https://cloud.githubusercontent.com/assets/233676/7860516/c69601ba-0541-11e5-8326-e577d34c243c.png)
![screen shot 2015-05-28 at 13 58 48](https://cloud.githubusercontent.com/assets/233676/7860515/c6953334-0541-11e5-8144-9edf7794f9f7.png)

alextea commented:
@tijmenb may I have a look at this in person? I am a little bit concerned about the text running for the full width of the page as this affects readability, and also I want to see how it looks at smaller page widths. Thanks

edds commented on app/assets/stylesheets/views/_topics.scss:
```diff
@@ -1,5 +1,12 @@
+  .topic-beta-label .govuk-beta-label span {
+    width: 75%;
+    vertical-align: top;
+  }
```
Ideally we shouldn't be styling the internals of a component within an application. If we did then we won't be able to iterate the component reliably without things breaking. Is there a way this could be incorporated into the components stylesheet?

alextea replied:
If that's the case I might pick this up and see if I can get the banner to work better with multiline text @edds @tijmenb

tijmenb replied:
Cool, @alextea is going to try to make it multiline, as it's already broken in `govuk_frontend_toolkit`:

http://govuk-component-guide.herokuapp.com/component/beta_label

![screen shot 2015-05-28 at 15 10 18](https://cloud.githubusercontent.com/assets/233676/7862078/acf29bf6-054b-11e5-8970-36ca3b24a656.png)

Then we don't need this extra styling anymore.

rboulton commented:
Apart from the styling issues being discussed, this looks good to me. :+1:

tijmenb commented:
I've removed the CSS from this branch and rebased, so the styling is currently broken. This can be merged after we've made the appropriate updates to https://github.com/alphagov/govuk_frontend_toolkit and https://github.com/alphagov/static.

rboulton commented:
Closing this pending changes to `govuk_frontend_toolkit` and `static`, since deploying it right now would break styling on the site.
